package.sh

```diff
@@ -4,10 +4,10 @@ rm -rf node_modules
 yarn install --production
 rm -f SHA256SUMS
 sha256sum package.json *.js LICENSE > SHA256SUMS
-find node_modules/ -type f -exec sha256sum {} \; >> SHA256SUMS
+find node_modules -type f -exec sha256sum {} \; >> SHA256SUMS
 TARFILE=$(npm pack)
-tar xf ${TARFILE}
+tar xzf ${TARFILE}
```

Comment:
The z is optional here, but not on czf - good catch

Reply:
The travis script on the addon-list repo caught it for me. :-)